stable/oauth2-proxy/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: oauth2-proxy
-version: 0.11.0
+version: 0.11.1
```

Comment:
Adding a new feature requires to bump the minor version to 0.12.0

Reply:
updated